tests/FSharp.UMX.Tests.fs

```diff
@@ -33,6 +34,7 @@ let ``Simple unit of measure conversions with cast operator``() =
     let w : string<foo> = % sprintf "%O %s %d" %x %y %z
     let b : byte<foo> = % 1uy
     let s : int16<foo> = % 1s
+    let f : float32<foo>  = % 10.0f
```

Comment:
Might make sense to add something that validates other common aliases work as well
```suggestion
    let f : float32<foo>  = % 10.0f
    let f : single<foo> = % 10.0f
```